src/main/java/com/sourcery/gymapp/backend/userProfile/service/UserProfileService.java

```diff
@@ -0,0 +1,61 @@
+package com.sourcery.gymapp.backend.userProfile.service;
+
+import com.sourcery.gymapp.backend.userProfile.dto.UserProfileDto;
+import com.sourcery.gymapp.backend.userProfile.exception.UserNotFoundException;
+import com.sourcery.gymapp.backend.userProfile.exception.UserProfileNotFoundException;
+import com.sourcery.gymapp.backend.userProfile.model.UserProfile;
+import com.sourcery.gymapp.backend.userProfile.repository.UserProfileRepository;
+import com.sourcery.gymapp.backend.userProfile.mapper.UserProfileMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class UserProfileService {
+
+    private final AuditorAware<UUID> auditorAware;
+    private final UserProfileRepository userProfileRepository;
+    private final UserProfileMapper userProfileMapper;
+
+    public UserProfileDto getUserProfile() throws UserNotFoundException, UserProfileNotFoundException {
+
+        UserProfile userProfile = userProfileRepository
+                .findUserProfileByUserId(
+                        auditorAware
+                                .getCurrentAuditor()
+                                .orElseThrow(UserNotFoundException::new));
+
+        if (userProfile==null)
+        {
```

Comment:
A minor thing but let's stick to one aproach when:
```
if (cond) {

}
```
not
```
if (cond) 
{

}
```

Reply:
Same question here with null check after `.orElseThrow(UserNotFoundException::new));`

Reply:
![image](https://github.com/user-attachments/assets/4d8d8bc2-33a0-4dd5-bb9c-6ace1419dec3)